.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+If related to user experience, use the format:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+If related to development, please provide relevant configuration details necessary to understand your problem including any relevant traces, logs, or otherwise.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```

Comment:
`Actual behavior` section ?

Reply:
Added in 3e2d3ce2a47db5d1c61b992e3b85017ddea6a610